components/Astronaout.jsx

```diff
@@ -0,0 +1,18 @@
+import Lottie from 'react-lottie';
+import React from 'react';
+import animationData from '../ lotties/astronaut.json';
+
+const defaultOptions = {
+  loop: true,
+  autoplay: true,
+  animationData: animationData,
+  rendererSettings: {
+    preserveAspectRatio: 'xMidYMid slice',
+  },
+};
+
+function AstronautLottie() {
+  return <Lottie height={400} options={defaultOptions} width={400} />;
+}
+
+export default AstronautLottie;
```

Comment:
We can simplify this further by combining lines 18 and 14 by doing the following

```suggestion
export default function AstronautLottie() {
  return <Lottie height={400} options={defaultOptions} width={400} />;
}
```